docs/wiki/resources.md

```diff
@@ -90,36 +89,31 @@
    
 ## Medium
 
-    [Grin's Mythical Fair Launch](https://www.coindesk.com/coders-harry-potter-bitcoin-mimblewimble) </br>
-    [Grin Money Explained](https://medium.com/@CryptoProfG/grin-money-explained-4-exploring-grins-monetary-model-e48b1761653) </br>
+:   [Grin Money Explained](https://medium.com/@CryptoProfG/grin-money-explained-4-exploring-grins-monetary-model-e48b1761653) </br>
     [Grin Transactions Explained, Step-by-Step](https://medium.com/@brandonarvanaghi/grin-transactions-explained-step-by-step-fdceb905a853) </br>
     [What’s inside a Grin Transaction File?](https://medium.com/@brandonarvanaghi/whats-inside-a-grin-transaction-file-f062a0dcbf99)</br>
     [Breaking Mimblewimble’s Privacy Model](https://medium.com/dragonfly-research/breaking-mimblewimble-privacy-model-84bcd67bfe52)</br>
     [Factual inaccuracies of “Breaking Mimblewimble’s Privacy Model”](https://medium.com/grin-mimblewimble/factual-inaccuracies-of-breaking-mimblewimbles-privacy-model-8063371839b9)</br>
     [Mimblewimble Without the Scary Math](https://blog.qtum.org/mimblewimble-without-scary-math-e894cb841b98) </br>
     [Behind Mimblewimble](https://medium.com/scalar-capital/behind-mimblewimble-cd9da78a00e9) </br>
-    [An Introduction to Grin Proof-of-Work](https://medium.com/blockcypher-blog/an-introduction-to-grin-proof-of-work-103aaa9f66ce) </br>
+    [An Introduction to Grin Proof-of-Work](https://blog.blockcypher.com/an-introduction-to-grin-proof-of-work-103aaa9f66ce) </br>
     [A Case for using soft total supply](https://john-tromp.medium.com/a-case-for-using-soft-total-supply-1169a188d153)
-    
 
 ## Podcasts
 
-    [[Bitcoin Wednesday] Introducing Mimblewimble and Grin @jaspervdm](https://www.youtube.com/watch?v=mzHswLujMYc) </br>
-    [[Unchained] Grin: A More Private, Lighter Bitcoin @lehnberg @yeastplume](https://unchainedpodcast.com/grin-a-more-private-lighter-bitcoin/) </br>
+:   [[Bitcoin Wednesday] Introducing Mimblewimble and Grin @jaspervdm](https://www.youtube.com/watch?v=mzHswLujMYc) </br>
+    [[Unchained] Grin: A More Private, Lighter Bitcoin @lehnberg @yeastplume](https://unchainedcrypto.com/grin-a-more-private-lighter-bitcoin/) </br>
     [[The Crypto Show] Mimblewimble with Andrew Poelstra and Peter Wuillie](https://soundcloud.com/heryptohow/mimblewimble-andrew-poelstra-peter-wuille-brian-deery-and-chris-odom) </br>
-    [[What Bitcoin Did] Grin's Mimblewimble Implementation @yeastplume](https://www.whatbitcoindid.com/podcast/grins-michael-cordner-aka-yeastplume-on-implementing-mimblewimble) </br>
     [[Zero Knowledge] Grin @lehnberg](https://fireside.fm/s/3yp1oIzN+6BedPBky) </br>
     [[Captain Crypto Show] Grin @yeastplume](https://www.youtube.com/watch?v=nwi9pMqUBQI) </br>
-    [[Let's Talk Bitcoin] Privacy with Mimblewimble @yeastplume @andreas @adam.b.levine](https://letstalkbitcoin.com/blog/post/lets-talk-bitcoin-356-privacy-on-the-blockchain-with-mimblewimble)
```

Comment:
There is this alternate link https://play.fountain.fm/episode/5Qk589TqsHFFx2W6F1uZ although the audio doesn't seem to play

Reply:
I added alternative link.